README.md

```diff
@@ -1 +1,92 @@
-# SOURCE CONTROL HANDS ON WORKSHOP
\ No newline at end of file
+# SOURCE CONTROL HANDS ON WORKSHOP
+
+## Table of content
+
+* [Slide](#Slide)
+* [Overall Git's structure](#Overall-Git's-structure)
+* [Remote VS Local](#Remote-VS-Local)
+* [Untracked files](#Untracked-Files)
+* [Tracked Files](#Tracked-Files)
+
+## Slide
+https://docs.google.com/presentation/d/1cCGnm1pLVwWVn6EGceNRsQSW8wvKztiTQKo3kP4yGHU/edit?usp=sharing
+
+### Key words from the slide
+* Source Control
+* Git
+* GitHub
+* Fork
+* Clone
+
+Some of the key words that you have gone through from the slide above might not yet been explained. But at the end of the workshop, I can promise, it will be crystal clear to you.
+
+## Overall Git's structure
+We now know that the practice of using source control is very beneficial to the project. However, it would not be a clever idea if you have to create your own version control system. And this is where Git shines.
+
+### Remote VS Local
+Git uses `Distributed Version Control Systems` as it structure so the history of your project is not only stored on the server, but on the local machine as you have clone the project. At the end of the presentation slide, you visited your repository on GitHub's website. That was where your remote history of the project reside. Later, you used the command `git clone <HTTPS or SSH link>` and that was when the history of the project is being copied from the *remote* server in to your *local* machine.
+
+The advantage of Distributed Version Control Systems that Git uses is that all of the team members have their own *full* history of the project. If the server dies, or lost the project's history, any of the member local project can be copied back to the server without any lost. 
+
+### Work on the local space
+Next step, let's add a new file into the project. On this current directory add a new file called `README2.md`. Under that file, just simple type you name into the file. Now go to `terminal` and type `git status` and you will see the following.
+
+![untracked-file](https://user-images.githubusercontent.com/11821799/51422310-00684080-1bdf-11e9-8285-59fdedd4425e.png)
+
+#### Untracked Files
+
+Since `README2.md` is newly created, for sure, Git will not see it in the latest snapshot history of the project so Git puts `README2.md` into untracked state.
```

Comment:
We talk about **snapshot** here but we haven't explain it in prior sections. I think we should explain the concepts of Git in **Overall Git's structure** like explaining that Git is actually a miniature file system that keeps snapshots but in simple terms .. maybe with some analogy, to go with the example in the slide where we had to name zip files uglily.

I'll help! need to sit down and think a bit.

Reply:
Very good read. Need to digest. Haha  
https://git-scm.com/book/en/v2/Git-Internals-Git-Objects